Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -617,18 +617,23 @@ def upload_file(self, ticket_id: str, file_id: str, file_name: str, ticket_type:
         return self.send_request('attachment/upload', 'POST', headers={'Accept': 'application/json'},
                                  body=body, file={'id': file_id, 'name': file_name})
 
-    def query(self, table_name: str, sys_param_limit: str, sys_param_offset: str, sys_param_query: str) -> dict:
-        """Query tickets by sending a PATCH request.
+    def query(self, table_name: str, sys_param_limit: str, sys_param_offset: str, sys_param_query: str,
+              system_params: str = '') -> dict:
+        """Query records by sending a GET request.
 
         Args:
         table_name: table name
         sys_param_limit: limit the number of results
         sys_param_offset: offset the results
         sys_param_query: the query
+        system_params: system parameters
 
         Returns:
             Response from API.
         """
+        if system_params:
```

Comment:
Why not append them to the query params below?

Reply:
Because we said that we want to override all the system params e.g offset, limit and. query. no?

Reply:
If so, the params need to be sent as dict to the request.